SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
Thanks, I checked the new changes but still don’t see completely through. I also do not see any exclude, I just see contains, maybe it is too early in the morning for me  😬  

The general logic should be:
1. Log everything that is on the same log level or above the min level
2. Additionally log things that are added as filters with `contains`
3. Do not log things that are in an exclusion filter (`exclude`), even if they are on the same or higher log level

Makes sense?